openghg/standardise/emissions/_intem.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Dict, Literal, Optional, Union
+
+
+def parse_intem(
+    filepath: Path,
+    species: str,
+    source: str = "intem",
+    domain: str = "europe",
+    data_type: str = "emissions",
+    database: Optional[str] = None,
+    database_version: Optional[str] = None,
+    model: Optional[str] = None,
+    period: Optional[Union[str, tuple]] = None,
+    chunks: Union[int, Dict, Literal["auto"], None] = None,
+    continuous: bool = True,
+) -> Dict:
+    """
+    Parse INTEM emissions data from the specified file.
+
+    Args:
+        filepath (Path): Path to the '.nc' file containing INTEM emissions data.
+        species (str): Name of species
+        source (str): Source of the emissions data, default is 'intem'.
+        domain (str): Geographic domain, default is 'europe'.
+        data_type (str): Type of data, default is 'emissions'.
+        database (Optional[str]): Database name if applicable.
+        database_version (Optional[str]): Version of the database if applicable.
+        model (Optional[str]): Model name if applicable.
+        period : "needs explainer"
+        chunks : (Union[int, Dict, Literal["auto"], None]): Chunking configuration.
+        continuous (bool): "needs explainer"
+
+    Returns:
+        Dict: Parsed emissions data in dictionary format.
+    """
+    from openghg.util import timestamp_now
+    from openghg.store import infer_date_range
+    from openghg.store._emissions import Emissions
+    from xarray import open_dataset
+
+    emissions_dataset = open_dataset(filepath, chunks=chunks)
```

Comment:
```suggestion
    emissions_dataset = open_dataset(filepath).chunk(chunks)
```

Reply:
xarray often doesn't chunk files correctly if you use the chunks argument to `open_dataset`, it's best to chunk after opening and chain the calls.